firefox-ios/WidgetKit/DownloadManager/DownloadLiveActivity.swift

```diff
@@ -5,9 +5,52 @@
 import WidgetKit
 import ActivityKit
 import SwiftUI
+import Foundation
 
 struct DownloadLiveActivityAttributes: ActivityAttributes {
     struct ContentState: Codable, Hashable {
+        struct Download: Codable, Hashable {
+            var id: UUID
+            var fileName: String
+            var mimeType: String
+            var hasContentEncoding: Bool?
+            var downloadPath: URL
+
+            var totalBytesExpected: Int64?
+            var bytesDownloaded: UInt64
+            var isComplete: Bool
+        }
+        var downloads: [Download]
+
+        func getCompletedDownloads() -> Int {
```

Comment:
```suggestion
        var completedDownloads: Int {
```
I suggest we make this a simple computed variable in Swift since it's just a value.